BazanovaNadezhda/headsAndTails.js

```diff
@@ -0,0 +1,50 @@
+const minimist = require('minimist');
+const readline = require('readline');
+const fs = require('fs');
+
+const argv = minimist(process.argv.slice(2));
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+
+const fileNameLog = argv._.toString();
+console.log(fileNameLog);
+
+console.log('Угадайте какой стороной упадёт монета.\n 1(орёл)\n 2(решка)\n q - выход из игры.\n');
+
+rl.on('line', (cmd) => {
+    if (cmd === 'q') {
+        rl.close();
+    }
+    else {
+        let coin=Math.round(Math.random()) + 1;
```

Comment:
обращайте внимание на оформление кода
```suggestion
        let coin = Math.round(Math.random()) + 1;
```